health-sdk/sdk/src/main/java/net/gini/android/health/sdk/integratedFlow/PaymentFragment.kt

```diff
@@ -82,7 +82,14 @@ data class PaymentFlowConfiguration(
      *
      * Note: Only taken into consideration for payments initiated with [documentId]
      */
-    val showCloseButtonOnReviewFragment: Boolean = false
+    val showCloseButtonOnReviewFragment: Boolean = false,
+    /**
+     * Defines the duration (in seconds) for which the payment review popup will be displayed before automatically closing.
+     * This value should be within a reasonable range (e.g., 0-10 seconds) to ensure a smooth user experience.
+     *
+     * Default value is `3` seconds.
+     */
+    val popupDurationPaymentReview: Int = 3
```

Comment:
[Request] Please have `3` as a constant, so we only need to change it in one place if needed (it's also used further down).